### DISCUSSION TITLE
随手写了个bat文件自动运行模拟器

### DISCUSSION THREAD

**Discussion by Youngmoss** [MAA1999/M9A#252] [Show and tell]
@echo off
echo Starting MuMu emulator with argument...
start "" "C:\Program Files\Netease\MuMuPlayer-12.0\shell\MuMuPlayer.exe" -v 2
echo Waiting for 5 seconds...
ping 127.0.0.1 -n 6 > nul
echo Starting M9A_CLI.exe...
start "" "%~dp0M9A_CLI.exe"
echo All tasks started.
exit
放在m9a_cll.exe同目录， "C:\Program Files\Netease\MuMuPlayer-12.0\shell\MuMuPlayer.exe" -v 2 可以通过模拟器的快捷方式->属性->目标获取自己的，要是电脑启动模拟器很满可以调大ping 127.0.0.1 -n 6 > nul的值‘6’

**Comment by SKIE1** [comment]
可以增加在运行完毕后关闭模拟器吗
